### DISCUSSION TITLE
继cmd在打开聊天窗口打印一次历史好友申请记录之后，这次cmd又在机器人不管是群主还是管理员的情况下，对同一次入群申请同时打印两次的情况

### DISCUSSION THREAD

**Discussion by STESmly** [nonebot/discussions#493] [问答]
![96c28dcb7b1d5281b283df0ece8d66e2](https://github.com/user-attachments/assets/b393b62f-1625-41e9-9ccc-35cd56d172a8)
![c8d315f423ec265feb116da3c23312b4](https://github.com/user-attachments/assets/2e8f75ff-218f-4181-a3b7-d1e04a833f1f)
同样的，这种情况也是发生在使用llob重装QQnt插件之后

**Comment by yanyongyu** [comment]
打印两次就是收到了协议端的两次事件，你应该去询问协议端。
